source/draw/gpu/opengl/backend/EGLImage.ooc

```diff
@@ -32,9 +32,14 @@ EGLImage: class extends GLTexture {
 		super()
 	}
 	bindSibling: func {
+		version(debugGL) { validateStart("EGLImage eglCreateImageKHR") }
 		eglImageAttributes := [EGL_IMAGE_PRESERVED_KHR, EGL_FALSE, EGL_NONE] as Int*
 		this _eglBackend = This _eglCreateImageKHR(this _eglDisplay, EGL_NO_CONTEXT, EGL_NATIVE_BUFFER_ANDROID, this _nativeBuffer, eglImageAttributes)
+		version(debugGL) { validateEnd("EGLImage eglCreateImageKHR") }
+		raise(this _eglBackend == EGL_NO_IMAGE_KHR, "EGL_NO_IMAGE_KHR returned")
```

Comment:
I guess I should ask whether `Debug error` is useful here as well or if `raise` is good enough.